### PR TITLE
Add #10 New gist about commiting from a user pool added

### LINKED ISSUE #10: Git commit from user pool
Could someone please help migrate this over to the instructed markdown format which this repository follows

**Description:**

     Makes things a little bit faster if you were to commit as multiple users to a single repository

**Usage:**

     - sh cfup "chore: some commit message" - Commits and pushes as a random user from the pool
     
     - sh cfup "chore: some commit message" shelob - Commits and pushes as user shelob
     
**Script:**

```bash
#!/bin/bash

message=$1
user=$2

configure_commit_push() {
   git config user.name "$1"
   git config user.email "$2"
   git add . && git commit -m "$message" && git push
}

talion() {
    configure_commit_push "Talion" "talion@gmail.com"
}

celebrimbor() {
    configure_commit_push "Celebrimbor" "celebrimbor@gmail.com"
}

shelob() {
    configure_commit_push "Shelob" "shelob@gmail.com>"
}

functions=("talion" "celebrimbor" "shelob")

if [ -z "$user" ]; then
    random_function=${functions[$RANDOM % ${#functions[@]}]}
    $random_function
elif type "$user" >/dev/null 2>&1; then
    $user
else
    echo "Invalid user provided."
fi
```

### PR DESCRIPTION
**Pull request by ThulinaWickramasinghe**

### CODE REVIEW

File: user-pool-commits.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+talion() {
+    configure_commit_push "Talion" "talion@gmail.com"
+}
+
+celebrimbor() {
+    configure_commit_push "Celebrimbor" "celebrimbor@gmail.com"
+}
+
+shelob() {
+    configure_commit_push "Shelob" "shelob@gmail.com>"
```
Comment (GIHAA) [inline user-pool-commits.md]: There is a small typo here ".com>"

File: user-pool-commits.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+- User should have git installed on their machine
+
+## Steps to Setup
+
+1. **Add the following bash script to the folder containing cloned repo. Recommended file name is 'cfup.sh':**
+
+```bash
+#!/bin/bash
+
```
Comment (GIHAA) [inline user-pool-commits.md:17]: let's add some error handling here to check weather user have git installed.. 
```
if ! command -v git &>/dev/null; then
    echo "git is not installed. Please install git to use this script."
    exit 1
fi
```
what do you think?

**Review by GIHAA — CHANGES_REQUESTED** [review]

File: user-pool-commits.md (hunk 1 above)
Comment (ThulinaWickramasinghe) [reply user-pool-commits.md]: Fixed it

File: user-pool-commits.md (hunk 2 above)
Comment (ThulinaWickramasinghe) [reply user-pool-commits.md:17]: Someone without git installed won't be looking in GitHub for gists. But we'll add this just in case.

**Pull request merged**